lib/binding.ex

```diff
@@ -33,16 +37,20 @@ defmodule GenRMQ.Binding do
   @doc false
   def declare_exchange(_chan, :default), do: :ok
 
-  def declare_exchange(chan, {:direct, exchange}) do
-    Exchange.direct(chan, exchange, durable: true)
+  def declare_exchange(chan, {type, exchange}) do
+    declare_exchange(chan, {type, exchange, true})
   end
 
-  def declare_exchange(chan, {:fanout, exchange}) do
-    Exchange.fanout(chan, exchange, durable: true)
+  def declare_exchange(chan, {:direct, exchange, durable}) do
```

Comment:
I suggest we change the API for declare_exchange (in a backwards compatible way) so we can use all options supported by `amqp`. Maybe something like:

```elixir
def declare_exchange(chan, exchange, type \\ :topic, options \\ [durable: true])

def declare_exchange(_chan, :default, _type, _opts), do: :ok

# For backwards compatibility
def declare_exchange(chan, {type, exchange}, _type, _opts) do
  declare_exchange(chan, exchange, type, durable: true)
end

def declare_exchange(chan, exchange, type, options) do
  Exchange.declare(chan, exchange, type, options)
end
```

what do you think about it @mkorszun @lukyanov ?